fmt: panic when formatting code

```sh
fn aws_instance_create(imgid, kname, secgrps, type, privip, subnetid) {
	instid <= (
		aws ec2 run-instances
				--image-id
				$imgid --count 1 --key-name kubernetes --security-group-ids $secgrps --instance-type $type --private-ip-address $privip --subnet-id $subnetid
                                				--associate-public-ip-address |
		jq -r ".Instances[].InstanceId"
	)
}
```

generated the following panic when formatted with nashfmt:

```sh
nash-fmt errors:
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x4c9ea0, 0x55fbc0)
	/home/i4k/projects/personal/go/src/runtime/panic.go:500 +0x1a0
github.com/NeowayLabs/nash/ast.(*CommandNode).toStringParts(0xc420056120, 0x2, 0x2, 0xc420048400, 0x2)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:249 +0x4d1
github.com/NeowayLabs/nash/ast.(*PipeNode).multiString(0xc420048280, 0xc420037a60, 0x40df20)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:345 +0xa6
github.com/NeowayLabs/nash/ast.(*PipeNode).string(0xc420048280, 0x8, 0x10000c420037ab0, 0x7f2cc93b6000)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:395 +0x141
github.com/NeowayLabs/nash/ast.(*ExecAssignNode).string(0xc4200482c0, 0x50, 0x0, 0xc420037a00)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:202 +0x248
github.com/NeowayLabs/nash/ast.(*ExecAssignNode).String(0xc4200482c0, 0x0, 0x2000)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:218 +0x2b
github.com/NeowayLabs/nash/ast.(*BlockNode).String(0xc42000a7b0, 0xc42004c190, 0x48)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:107 +0xf1
github.com/NeowayLabs/nash/ast.(*Tree).String(0xc42000e2c0, 0xc42004c140, 0x44)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/tree.go:35 +0x41
github.com/NeowayLabs/nash/ast.(*FnDeclNode).String(0xc42004c0f0, 0xc4200016c0, 0x25)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:538 +0x148
github.com/NeowayLabs/nash/ast.(*BlockNode).String(0xc42000a450, 0x0, 0x0)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/node_fmt.go:107 +0xf1
github.com/NeowayLabs/nash/ast.(*Tree).String(0xc42000e360, 0x0, 0x0)
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/ast/tree.go:35 +0x41
main.main()
	/home/i4k/projects/go-workspace/src/github.com/NeowayLabs/nash/cmd/nashfmt/main.go:68 +0x1cb
```

--- a/ast/node_fmt.go
+++ b/ast/node_fmt.go
@@ -246,7 +246,7 @@ func (n *CommandNode) toStringParts() ([]string, int) {
 				line = ""
 			}
 
-			if next[0] != '-' {
+			if len(next) > 0 && next[0] != '-' {
 				if line == "" {
 					line += arg + " " + next
 				} else {

